src/main/webapp/business.js

```diff
@@ -1,42 +1,70 @@
-// implementing map
+function getInfo() {
+  fetch('/business').then(response => response.json()).then((json) => {
+    var counter = 0;
+    for (var i = 1; i < json.length; i++) {
+      if (json[i][1] == 'in-person') {  // makes list of in-person businessess
```

Comment:
Put the comment above the code clock or remove it if you move the code to a function

Reply:
changed